lk2nd/device/dts/msm8974/rules.mk

```diff
@@ -8,6 +8,7 @@ QCDTBS += \
 	$(LOCAL_DIR)/msm8974-htc-m8.dtb \
 	$(LOCAL_DIR)/msm8974-lge-d855.dtb \
 	$(LOCAL_DIR)/msm8974-lge-hammerhead.dtb \
+	$(LOCAL_DIR)/msm8974-samsung-hlte.dtb \
```

Comment:
Hm, would you mind adding the device and the new msm-id to the common samsung.dts? For the new lk2nd we decided to squash all the rev files together since it usually causes a mess otherwise...

Reply:
Hmm... That's what happens when I do that.
[lk2nd.log](https://github.com/user-attachments/files/16202790/lk2nd.log)
(Unable to boot anything in state, as you can guess from this log.)

Maybe it's still worth keeping different models in different files? Note 3's variants in one, S5's in other? After all, there's still some difference between Pro and non-Pro variant of MSM8974.

Reply:
If you want to make sure lk2nd can boot downstream, you need to add override values in the device node, that is, you'd add `qcom,msm-id = <0x7E01FF01 7 0>;` (or whatever is correct for your downstream) into the hlte node, then lk2nd will use that value to discover the dtb in downstream images.

Reply:
Alright, I'll try to do so, then.

Reply:
Well, that just works.